crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -684,6 +680,19 @@ fn handle_deploy_result(
     }
 }
 
+fn serialize_call_trace(call_trace: &CallTrace) -> Vec<Felt252> {
```

Comment:
Can we put this as an `impl` of `CallTrace` in it's file?

Reply:
I don't really agree, we cannot implement it for CallEntryPoint bcs it is an external struct, so it is cleaner to have both implemented as functions. It doesn't really change much.